### ISSUE TITLE
filtering in custom taxonomy call

### ISSUE THREAD

**Issue by denisyilmaz** [svrooij/rest-api-filter-fields#6]
Plugin is working great, even with categories etc. But I cant get it to work for custom taxonomies:

Example:
http://www.site.com/wp-json/wp/v2/custom-taxonomy?fields=id,name,count

```
add_action( 'init', 'create_custom_tax', 0);

function create_custom_tax() {
	$labels = array(…);

	$args = array(
		'hierarchical'      => true,
		'labels'            => $labels,
		'show_ui'           => true,
		'show_admin_column' => true,
		'query_var'         => true,
		'rewrite'           => array( 'slug' => 'custom_tax' ),
		'show_in_rest'       => true,
  		'rest_base'          => 'custom-taxonomy',
  		'rest_controller_class' => 'WP_REST_Terms_Controller'
	);

	register_taxonomy( 'custom_tax', array( 'post' ), $args );
}
```

… just gives the taxonomy result, but no filtering by the fields. Tested in Wordpress 4.7.2
any ideas?
Thanks a lot!

**Comment by denisyilmaz** [comment]
ok, found a workaround:

when adding `add_filter('rest_prepare_custom_tax',array($this,'filter_magic'),20,3);` in _class-rest-api-filter-fields.php_ it's working.

I think the plugin needs a loop through all tax_types as its doing for post_types?

**Comment by denisyilmaz** [comment]
i added a loop for custom taxonomies: see #7

**Comment by svrooij** [comment]
Just released to plugin directory. Thanks for the fix, (you're in the release notes [V1.0.6](https://github.com/svrooij/rest-api-filter-fields/blob/master/README.md#106)
